agent.go

```diff
@@ -63,9 +64,13 @@ type Payload struct {
 
 // ProbeResult structure for network probing results
 type ProbeResult struct {
```

Comment:
Not for the agent, I suppose. It would mean we check connectivity with k8s API server rather than with netchecker-server. It would change the original design of netchecker. I'd probably add an issue about this. I'm not sure we need this though.
And there is no sense to store it for long. It is updated with report_interval.
But for the server, corresponding gauges could be stored in TPR along with other metrics.